package.json

```diff
@@ -68,8 +68,8 @@
     "profile": "REPORT_GAS=true hardhat test",
     "precoverage": "cp .openzeppelin/avalanche.json .openzeppelin/unknown-31337.json",
     "coverage": "hardhat coverage --config ./hardhat-coverage.config.ts --solcoverjs ./.solcover.js",
-    "lint:fix": "prettier --write '**.sol' && prettier --write '**.js' && prettier --write '**.ts'",
-    "lint:check": "prettier --check '**.sol' && prettier --check '**.js' && prettier --check '**.ts' && hardhat compile && npx solhint --config ./.solhint.json --ignore-path ./.solhintignore 'contracts/**/*.sol'",
+    "lint:fix": "prettier --write '**.sol' && prettier --write '**.js' && prettier --write '**.ts' && prettier --check '**.json'",
```

Comment:
I think fix and check tasks should not be mixed: can you please remove the check for lint:fix?

Reply:
Oh it should be --write for lint:fix. Thanks.